tools/appveyor.psm1

```diff
@@ -108,6 +108,20 @@ function Invoke-AppVeyorInstall
         Update-AppveyorBuild -message $buildName
     }
 
+    # Generate credentials for appveyor remoting tests.
```

Comment:
The guidance for enabling rdp in appveyor is to run this script:

https://raw.githubusercontent.com/appveyor/ci/master/scripts/enable-rdp.ps1

it will set the password and display it in the logs so you can log in.  I needed this functionality yesterday, so I want to make sure we can keep doing that.

I think this change would break that.  Maybe that's fine, but we should add guidance like in this doc:
https://www.appveyor.com/docs/how-to/rdp-to-build-worker/